parsers/rpmspec.c

```diff
@@ -154,7 +154,7 @@ static void found_package_cb (const char *line,
 		vString *name = vStringNew ();
 		tagEntryInfo tag;
 
-		vStringNCopyS (name, line + matches[1].start, matches[1].length);
+		vStringNCopyS (name, line + matches[2].start, matches[2].length);
```

Comment:
You are correct. Thank you for spotting this. I will do "git push --force" after rearranging the commits.